.refactored/sciserver-rstudio/2.0/Dockerfile

```diff
@@ -0,0 +1,124 @@
+FROM sciserver-base-texlive
+
+RUN yum -y update
+RUN yum -y install epel-release
+RUN yum -y install \
+                gcc \
+                gcc-gfortran \
+                zeromq-devel \
+                zlib-devel \
+                readline-devel \
+                bzip2-devel \
+                xz-devel \
+                pcre-devel \
+                openssl-devel \
+                libcurl-devel \
+                libpng-devel \
+                libtiff-devel \
+                libjpeg-devel \
+                cairo-devel \
+                libXt-devel
+
+RUN yum config-manager --set-enabled powertools
+RUN yum -y install \
+                texinfo \
+                pandoc \
+                glibc-locale-source \
+                glibc-langpack-en \
+                initscripts
+
+RUN wget -nv https://download2.rstudio.org/server/centos8/x86_64/rstudio-server-rhel-1.4.1103-x86_64.rpm \
+        && yum install -y rstudio-server-rhel-*.rpm \
+        && rm rstudio-server-rhel-*.rpm \
+        && yum clean all \
+        && rm -rf /var/cache/yum
+
+
+ENV PATH /home/idies/R/bin:$PATH
+
+USER idies
+
+ARG R_VERSION=4.0.3
+
+RUN cd ~ \
+        && curl -L https://cran.rstudio.com/src/base/R-4/R-$R_VERSION.tar.gz -o R.tar.gz \
+        && tar -xzf R.tar.gz \
+        && cd R-$R_VERSION \
+        && ./configure --prefix=/home/idies/R --enable-R-shlib --with-x=no \
+        && make -j`nproc` \
+        && make install \
+        && cd .. \
+        && rm -rf R.tar.gz R-$R_VERSION
+
+RUN Rscript -e "install.packages(c('Cairo'), repos='https://cloud.r-project.org/')"
+
+RUN /bin/bash -c $'printf "options(bitmapType=\'cairo\')\n" >> ~/.Rprofile'
+
+
+RUN mkdir -p ~/.rstudio/monitored/user-settings \
+        && echo 'initialWorkingDirectory=~/workspace' > ~/.rstudio/monitored/user-settings/user-settings
+
+USER root
+
+RUN yum -y install nginx
+
+COPY rserver.conf /etc/rstudio/rserver.conf
+
+COPY nginx.conf /etc/nginx/nginx.conf
+
+COPY entrypoint.sh /etc/entrypoint.sh
+
+COPY --chown=idies Rprofile /home/idies/.Rprofile
+
+## Disabled for compatibility with SciServer
+# ENTRYPOINT /etc/entrypoint.sh
+
+EXPOSE 8888
+
+
+## Added for compatibility with SciServer
+COPY entrypoint.sh /opt/startup.sh
```

Comment:
`entrypoint.sh` would be better named `startup.sh`, since it is not used as `ENTRYPOINT` in the dockerfile and rather is named startup.sh, would be easier to read the repository setup this way.

Reply:
yes, fixed.